packages/data-stores/src/automated-transfer-eligibility/reducer.ts

```diff
@@ -0,0 +1,25 @@
+import { combineReducers } from '@wordpress/data';
+import { State } from './types';
+import type { Action } from './actions';
+import type { Reducer } from 'redux';
+
+export const transferEligibility: Reducer< State, Action > = ( state = {}, action ) => {
+	switch ( action.type ) {
+		case 'TRANSFER_ELIGIBILITY_RECEIVE': {
+			return {
+				...state,
+				[ action.siteId ]: action.transferEligibility,
+			};
+		}
+	}
+
+	return state;
+};
+
+const reducer = combineReducers( {
+	transferEligibility,
+} );
```

Comment:
Now when there's only one sub-reducer, you don't need to combine it. Just make `transferEligibility` itself into the store reducer.

Reply:
`type State = ReturnType< typeof transferEligibility >` does this work?

Reply:
No, that didn't work. What was wrong here was trying to use the state as if it was combined in the reducer. I just removed the `RootState` type and everything worked :). Ref: https://github.com/Automattic/wp-calypso/pull/62861/commits/f6456d579b3f17b0aca1907cd0930b2a035ff1ca